app/Console/Commands/Permissions/DefaultsCommand.php

```diff
@@ -0,0 +1,153 @@
+<?php
+
+namespace App\Console\Commands\Permissions;
+
+use HMS\Entities\Role;
+use Illuminate\Console\Command;
+use Illuminate\Support\Facades\Artisan;
+use Doctrine\ORM\EntityManagerInterface;
+use LaravelDoctrine\ACL\Permissions\Permission;
+
+class DefaultsCommand extends BaseCommand
+{
+    /**
+     * The name and signature of the console command.
+     *
+     * @var string
+     */
+    protected $signature = 'permissions:defaults';
+
+    /**
+     * The console command description.
+     *
+     * @var string
+     */
+    protected $description = 'Restores roles and permissions to the default set. You should probably run migrations beforehand and the seeder afterwards';
+
+    /**
+     * @var EntityManagerInterface
+     */
+    private $em;
+
+    /**
+     * The permissions to set up
+     * @var array
+     */
+    private $permissions = [
```

Comment:
move $permissions and $roles out to a config/ file
dropping the use of Role constants for just strings